Doc/library/math.rst

```diff
@@ -224,9 +224,9 @@ Number-theoretic and representation functions
    of *x* and are floats.
 
 
-.. function:: nextafter(x, y)
+.. function:: nextafter(x, y, /, *, steps=1)
 
-   Return the next floating-point value after *x* towards *y*.
+   Return the floating-point value *steps* steps after *x* towards *y*.
```

Comment:
The "If *x* is equal to *y*, return *y*" text below is no longer valid in the case where `x = -0.0`, `y = 0.0` and `steps = 0`. I think the behaviour in the PR is the right one (`steps = 0` should match applying `nextafter` zero times), but we may need to tweak the docs.